install-win.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/sh
+set -e
+
+# Setup Python venv and install packages
+echo "Setting up Python virtual environment in ./.venv"
+python -m venv ./.venv
+. .venv/Scripts/activate
```

Comment:
Instead of two separate files, can you modify the original to act differently depending on system? This SO post might work: [How to detect the OS from a Bash script?](https://stackoverflow.com/a/8597411)